### ISSUE TITLE
Feature suggestion: line endings

### ISSUE THREAD

**Issue by zygimantas** [codecadwallader/codemaid#306]
### Environment
- Visual Studio version: 2015 any
- CodeMaid version:
- Code language: any

### Description
It would be nice to have an option to configure prefered line endings which should be used for replacement during the reorganization phase.

**Comment by codecadwallader** [comment]
Thanks for the suggestion.  I would think that would be a setting across all CodeMaid operations (not just reorganizing).  There was a card in our Trello backlog (which is now deprecated) similar to this: https://trello.com/c/vV8z9cSl/134-end-of-line-terminator-options but it's good to have one over in GitHub to see how much interest it garners.

**Comment by zygimantas** [comment]
Yes, it is very similar. **Here is the use case of my suggestion:**

I use a CLI tool to generate .html/.css/.ts files. It is one-time operation, but the tool generates it with LF endings. The only way, known to me, is to use "File->Advanced Save Options" and change the line-endings for each generated file. It would be good if CodeMaid could do that for me.

**Comment by kfenner** [comment]
Thanks for bringing this up :+1: 

This is a much needed feature since Visual Studio does not allow to configure the line endings for a project or globally. Now when using Visual Studio for a project which is using LF line endings, CodeMaid always inserts CRLF when reformatting the code. This is very annoying.
